napari/layers/surface/surface.py

```diff
@@ -291,6 +293,17 @@ def __init__(
     def _calc_data_range(self, mode='data'):
         return calc_data_range(self.vertex_values)
 
+    @property
+    def _contrast_limits_range(self):
+        if self._contrast_limits_range is not None:
+            return self._contrast_limits_range
+        return calc_data_range(self._vertex_values)
+
+    @_contrast_limits_range.setter
+    def _contrast_limits_range(self, value):
+        # If you want to allow setting the contrast_limits_range, you can define a setter method.
+        self._contrast_limits_range = value
+
```

Comment:
why this is required?

Reply:
There is an inconsistency in the use of 'contrast_limit_range' in the intensity Mixin and the image and surface layer. For some reason the Mixin '_contrast_limit_range' is not used in 'ImageBase' but is in the Surface layer. Leads to inconsistent typing. I implemented fix wrongly

Reply:
It is not used in ImageBase, as ImageBase uses property `contrast_limit_range` defined by minix.